src/modules/mavlink/streams/GPS_RAW_INT.hpp

```diff
@@ -56,14 +58,16 @@ class MavlinkStreamGPSRawInt : public MavlinkStream
 	explicit MavlinkStreamGPSRawInt(Mavlink *mavlink) : MavlinkStream(mavlink) {}
 
 	uORB::Subscription _sensor_gps_sub{ORB_ID(sensor_gps), 0};
+	hrt_abstime _last_send_ts {};
+	static constexpr hrt_abstime _no_gps_send_interval {1_s};
```

Comment:
We're slowly trying to get aligned with the google c++ style guide.

```suggestion
	static constexpr hrt_abstime kNoGpsSendInterval{1_s};
```